app/views/stories/feeds/show.json.jbuilder

```diff
@@ -0,0 +1,16 @@
+article_attributes_to_include = %i[
+  title path id user_id comments_count positive_reactions_count organization_id
+  reading_time video_thumbnail_url video video_duration_in_minutes language
+  experience_level_rating experience_level_rating_distribution cached_user
+  cached_organization main_image
+].freeze
+article_methods_to_include = %i[
+  readable_publish_date cached_tag_list_array flare_tag class_name
+  cloudinary_video_url video_duration_in_minutes published_at_int
+  published_timestamp
+].freeze
```

Comment:
I would have loved for these to go in `Article.rb`, but that file's size and our Rubocop linting rules prevent any additions to that file.

Reply:
If it really makes sense to be in there, can you add a comment like eslint does to disable the rule or do we not wanna go there?

Reply:
You can, it's 

```
# rubocop:disable Metrics/ClassLength
class Article < ApplicationRecord
  ...
end
# rubocop:enable Metrics/ClassLength
```

but tbh if this is the only place where we need them I prefer keeping them here.

Reply:
A note, as this is rendered each time the route is hit, I'm not sure if freeze is really meaningful

Reply:
Good catch Rhymes: the `.freeze` is a leftover Rubocop appeasement from when I had them as constants.